### DISCUSSION TITLE
Design contribution

### DISCUSSION THREAD

**Discussion by amandauiux** [Ivy-Apps/ivy-wallet#3220] [Design]
Hi there,
I would like to know if there is a way to contribute to this project as a UX/UI designer, I saw that you started working on a desktop version of the app, if you need help I'd like to contribute with that or even with your design system for the hole application.

**Comment by ILIYANGERMANOV** [comment]
Hey @amandauiux design contributions are welcome! @stefanilijev97 who is our product designer should provide more info on how we can do it. I imagine designers collaborating with contributors on GitHub issues that they find interesting (https://github.com/Ivy-Apps/ivy-wallet/issues) and providing Figma designs so developers can build beautiful and user-friendly UIs.

**Reply by amandauiux** [reply]
Thanks for the response @ILIYANGERMANOV.
@stefanilijev97 let me know if there's something I can jump in, or we can talk about it if you'd like, feel free to message me.
